tests/database/image/unit/test_image.py

```diff
@@ -41,13 +47,16 @@ def tearDown(self):
 
```

Comment:
It just doesnt return the names of the images it returns more stuff. I advice looking at the method to get better understanding